plugin/trino-delta-lake/src/test/java/io/trino/plugin/deltalake/BaseDeltaLakeConnectorSmokeTest.java

```diff
@@ -1973,10 +1975,8 @@ public void testHistoryTableWithDeletedTransactionLog()
 
             String tableLocation = getTableLocation(table.getName());
             // Remove first two transaction logs to mimic log retention duration exceeds
-            String key = tableLocation.substring(bucketUrl().length());
-            MinioClient minio = hiveMinioDataLake.getMinioClient();
-            minio.removeObject(bucketName, "%s/_delta_log/%020d.json".formatted(key, 0));
-            minio.removeObject(bucketName, "%s/_delta_log/%020d.json".formatted(key, 1));
+            deleteFile("%s/_delta_log/%020d.json".formatted(tableLocation, 0));
```

Comment:
Is there any reason we want to run this test with several storages? I feel `TestDeltaLakeConnectorTest` is sufficient.